contractsSol5/Dao/mock/MockMaliciousDAO.sol

```diff
@@ -7,17 +7,15 @@ contract MockMaliciousDAO is KyberDAO {
     constructor(
         uint _epochPeriod, uint _startTimestamp,
         address _staking, address _feeHandler, address _knc,
-        uint _maxNumOptions, uint _minCampDuration,
-        uint _defaultNetworkFeeBps, uint _defaultRewardBps, uint _defaultRebateBps,
+        uint _minCampDuration, uint _defaultNetworkFeeBps, uint _defaultRewardBps, uint _defaultRebateBps,
         address _admin
     ) KyberDAO(
         _epochPeriod, _startTimestamp,
         _staking, _feeHandler, _knc,
         _defaultNetworkFeeBps, _defaultRewardBps, _defaultRebateBps,
         _admin
     ) public {
-        MAX_CAMP_OPTIONS = _maxNumOptions;
-        MIN_CAMP_DURATION_SECONDS = _minCampDuration;
+        MIN_CAMPAIGN_DURATION_SECONDS = _minCampDuration;
```

Comment:
camp --> campaign
MIN_CAMPAIGN_DURATION_SECONDS if not const. should be camel cased

Reply:
Updated